ixofoundation/ixo-webclient
# chore(style): project overview styling updates

## Scope
https://nonaredteam.atlassian.net/browse/IXO-351

## Work Done
Updated the ProjectFounder section
Styled the general Project Overview elements

## Steps to Test
Make sure the Project page is styled as per the designs
You can use the following data to show the sections:
```
props.project.users = [
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Joyce Montgomery',
      role: 'Co-Founder & CEO',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Islam Naidoo',
      role: 'CTO',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Jonathan Mike',
      role: 'Designer',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
    {
      imageUrl: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
      name: 'Daniel Ingram',
      role: 'Co-Founder & CEO',
      linkedinLink:
        'https://ch.linkedin.com/company/ixofoundation?trk=public_profile_topcard_current_company',
      twitterLink: 'https://twitter.com/ixoworld',
    },
  ]

  props.project.founder = {
    name: 'Ixo foundation',
    email: '',
    countryOfOrigin: 'IN',
    shortDescription:
      'The ixo foundation researches and develops open-source protocols for sustainable development.',
    websiteURL: 'http://localhost:3000',
    logoLink: 'https://pds_pandora.ixo.world/public/smsal67t2pgk8ss96ss',
  }

  props.project.socialMedia = {
    facebookLink: 'http://localhost:3000',
    instagramLink: 'http://localhost:3000',
    twitterLink: 'http://localhost:3000',
    webLink: 'http://localhost:3000',
  }
```

## Screenshots
![image](https://user-images.githubusercontent.com/46312800/79872383-03f00600-83e6-11ea-9a08-75f3b976bce4.png)
